safe-global/safe-react
# fix: adjust notification centre styles

## What it solves
Design adjustments

## How this PR fixes it
The following has been adjusted:
- Background colours added to `'info'` and `'success'` snackbar variants
- A divider is now present under the header
- Dividers are present between each notification
- The 'Clear All' button has a hover effect and isn't visible when there are no notifications
- An image is now present when there are no notifications

## How to test it
Open the notification centre with no notifications present and observe the header divider and new image. Trigger > 1 notification and observe the dividers between each notifications.

## Screenshots
![notification centre](https://user-images.githubusercontent.com/20442784/174784578-219ab40c-9f19-4d35-a0f4-156a1f93f523.gif)
